.github/first-timers-issue-template.md

```diff
@@ -0,0 +1,44 @@
+### 🆕🐥☝ First Timers Only.
+
+This issue is reserved for people who never contributed to Open Source before. We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝
+
+[About First Timers Only](http://www.firsttimersonly.com/).
+
+### 🤔 What you will need to know.
+
+Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.
+
+### 📋 Step by Step
+
+- [ ] 👌 **Join the team**: Add yourself to a Jekyll affinity team.
+
+  Go to [teams.jekyllrb.com](https://teams.jekyllrb.com/) join a team that best fits your interests.
```

Comment:
maybe flesh this part out, e.g. add that the person will be invited into the organization, but that this step is also optional

Reply:
This step is necessary for them to be able to asign themselves to issues, right?

Reply:
oh is it? if so, then this is fine

Reply:
Ok, so a member of a team _can_ be assigned to an issue, but cannot assign themselves to an issue. I will update the wording to reflect this.